test/integration/features/profile.feature

```diff
@@ -89,3 +89,17 @@ Scenario: User should be able to switch between non existing profiles
       oc cli context could not changed for 'minishift'. Make sure the profile is in running state or restart if the problem persists.
       Profile 'minishift' set as active profile
       """
+Scenario: User should not be able to create profile with blank profile name
```

Comment:
Given that we cover this in unit tests, there is not much value here, but I guess for completeness sake.